lib/phoenix_pubsub/tracker.ex

```diff
@@ -27,9 +27,10 @@ defmodule Phoenix.Tracker do
       delta broadcasts have been sent. Defaults `10` (15s heartbeat)
     * `down_period` - The interval in milliseconds to flag a replica
       as down temporarily down. Default `broadcast_period * max_silent_periods * 2`
-      (30s down detection).
+      (30s down detection). Note: This must be at least 2x the `broadcast_period`.
     * `permdown_period` - The interval in milliseconds to flag a replica
       as permanently down, and discard its state.
+      Note: This must be at greater than the `down_period`.
```

Comment:
Looks like this could be missing the word "least":

"at least greater" instead of "at greater".